src/SDKs/Network/Management.Network/Generated/INetworkInterfaceLoadBalancersOperations.cs

```diff
@@ -19,15 +19,18 @@ namespace Microsoft.Azure.Management.Network
     using System.Threading.Tasks;
 
     /// <summary>
-    /// AvailablePrivateAccessServicesOperations operations.
+    /// NetworkInterfaceLoadBalancersOperations operations.
     /// </summary>
-    public partial interface IAvailablePrivateAccessServicesOperations
+    public partial interface INetworkInterfaceLoadBalancersOperations
```

Comment:
@number213 
nit: Does it have to be plural (LoadBalancerS)? or was it suppose to be INetworkInterfaceLoadBalancerOperations (LoadBalancerOperations)
Because below all model types getting returned are IPage<LoadBalancer>

Also why the interface is renamed differently than the interface implementing class
IAvailabelPrivateAccessService ==> INetworkInterfaceLoadBalancers (interface renaming)
AvailablePrivateAccessService ==> LoadBalancerNetworkInterface (class renaming)

Was this an oversight?